src/Quartz.Benchmark/DefaultThreadPoolBenchmark.cs

```diff
@@ -25,15 +25,15 @@ public void RunInThread_CompletedTask_MaxConcurrencyIsMaxValue_SingleThreaded()
     }
 
     [Benchmark(OperationsPerInvoke = 1_000_000)]
-    public void RunInThread_CompletedTask_MaxConcurrencyIsMaxValue_MultiThreaded()
+    public async Task RunInThread_CompletedTask_MaxConcurrencyIsMaxValue_MultiThreaded()
```

Comment:
maybe these quite safe changes for tests/benchmarks could be a separate PR (calling async versions)

Reply:
Ok, if I do that I need to add more ignore clauses to stop the warnings from becoming errors and blocking build.

Reply:
Well without the analyzer it wouldn't break the build? So my suggestions would be a separate PR for all test/example/benchmark code without analyzers so it should work the same.